tests/Support/Data/ObjectWithLabelsProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator\Tests\Support\Data;
+
+use Yiisoft\Validator\Label;
+use Yiisoft\Validator\LabelsProviderInterface;
+use Yiisoft\Validator\Rule\Number;
+use Yiisoft\Validator\Rule\Required;
+
+final class ObjectWithLabelsProvider implements LabelsProviderInterface
+{
+    #[Required(message: '{attribute} cannot be blank.')]
+    public string $name = '';
+
+    #[Number(min: 21, lessThanMinMessage: '{attribute} must be no less than {min}.')]
+    protected int $age = 17;
+
+    #[Number(max: 100)]
+    #[Label('test')]
```

Comment:
Need additional test for case when propert contain attribute `Label` and `getValidationPropertyLabels()` contain label for same property.